website/docs/cloud-docs/api-docs/stacks.mdx

```diff
@@ -0,0 +1,946 @@
+
+## Overview
+
+The scope of the API includes the following endpoints:
+
+| Method   | Path                                                              | Action                                                                                                     |
+|----------|-------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------|
+| `POST`   | `/stacks`                                                         | Call this endpoint to [create a stack](#create-a-stack).                                                   |
+| `GET`    | `/organizations/:organization_name/stacks`                        | Call this endpoint to [list existing stacks](#list-stacks).                                                |
+| `GET`    | `/stacks/:stack_id`                                               | Call this endpoint to [show a stack](#show-a-stack).                                                       |
+| `PATCH`  | `/stacks/:stack_id`                                               | Call this endpoint to [update a stack](#update-a-stack).                                                   |
+| `DELETE` | `/stacks/:stack_id`                                               | Call this endpoint to [delete a stack](#delete-stack).                                                     |
+| `POST`   | `/stacks/:stack_id/stack-configurations`                          | Call this endpoint to [create a stack configuration](#create-a-stack-configuration).                       |
+| `GET`    | `/stacks/:stack_id/stack-configurations`                          | Call this endpoint to [list stack configurations](#list-stack-configurations).                             |
+| `GET`    | `/stack-configurations/:stack_configuration_id`                   | Call this endpoint to [show a stack configuration](#show-stack-configuration).                             |
+| `GET`    | `/stacks/:stack_id/stack-configuration-summaries`                 | Call this endpoint to [list stack configuration summaries](#list-stack-configuration-summaries).           |
+| `POST`   | `/stacks/:stack_id/fetch-latest-from-vcs`                         | Call this endpoint to [fetch the latest stack configuration](#fetch-the-latest-stack-configuration).       |
+| `GET`    | `/stack-configurations/:stack_configuration_id/stack-diagnostics` | Call this endpoint to [list stack configuration diagnostics](#list-stack-configuration-diagnostics).       |
+| `GET`    | `/stack-configurations/:stack_configuration_id/upload-url`        | Call this endpoint to [fetch stack configuration upload url](#fetch-stack-configuration-upload-url).       |
+| `GET`    | `/stack-configurations/:stack_configuration_id/source-bundle`     | Call this endpoint to [fetch stack configuration source bundle](#fetch-stack-configuration-source-bundle). |
```

Comment:
Are we supposed to observe any particular order with regards to API actions? It may be nice to group everything together for readability - but I'm not sure if that's an official requirement or suggestion.

Reply:
I’m happy to follow your suggestion on how these APIs should be ordered, as I don’t have a strong preference. :) They’re currently grouped based on the corresponding JIRA tickets, but we can absolutely revisit the ordering once all the PRs are merged into a single one.